README.md

```diff
@@ -26,30 +26,32 @@ In this repository, you will find source codes, explanations, and other resource
-      - 2.5.9 [IntelliSense for CSS class names in HTML](https://marketplace.visualstudio.com/items?itemName=Zignd.html-css-class-completion)
-      - 2.5.10 [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
-      - 2.5.11 [Lorem ipsum](https://marketplace.visualstudio.com/items?itemName=Tyriar.lorem-ipsum)  *VScode(Emmet) already has this feature. Type lorem and number of word and hit tab. Example. lorem100*
-      - 2.5.12 (File Icon Theme) [Material Icon Theme](https://marketplace.visualstudio.com/items?itemName=PKief.material-icon-theme)
-      - 2.5.13 [Open in Browser](https://marketplace.visualstudio.com/items?itemName=techer.open-in-browser)
-      - 2.5.14 [Prettier - Code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
-      - 2.5.15 (File Icon Theme) [vscode-icons](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons)
-      - 2.5.16 (Theme) [Dracula Official](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula)
-      - 2.5.17 (Font) [Fira Code](https://github.com/tonsky/FiraCode/releases/download/6.2/Fira_Code_v6.2.zip)
-      - 2.5.18 Goto Settings > search "font family" > set font to "Fira Code" (without quote)
-      - 2.5.19 Goto Settings > search "font ligature" > open in json > set "editor.fontLigatures": true
+     - 2.5.1 [Auto Close Tag](https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-close-tag)
+     - 2.5.2 [Auto Rename Tag](https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-rename-tag)
+     - 2.5.3 [Bootstrap 4, Font awesome 4](https://marketplace.visualstudio.com/items?itemName=thekalinga.bootstrap4-vscode) or (for Bootstrap 5) [Bootstrap 5 & Font awesome Snippets](https://marketplace.visualstudio.com/items?itemName=HansUXdev.bootstrap5-snippets)
+     - 2.5.4 [Code Runner](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner)
+     - 2.5.5 [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
+     - 2.5.6 [Color Highlight](https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight)
+     - 2.5.7 [Git History](https://marketplace.visualstudio.com/items?itemName=donjayamanne.githistory)
+     - 2.5.8 [GitLense - Git supercharged](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+     - 2.5.9 [IntelliSense for CSS class names in HTML](https://marketplace.visualstudio.com/items?itemName=Zignd.html-css-class-completion)
+     - 2.5.10 [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
+     - 2.5.11 [Lorem ipsum](https://marketplace.visualstudio.com/items?itemName=Tyriar.lorem-ipsum) _VScode(Emmet) already has this feature. Type lorem and number of word and hit tab. Example. lorem100_
```

Comment:
Use **

Reply:
Ok, I will use ** next time. Thanks a lot for your suggestion.